nrfs/include/internal/services/nrfs_gdpwr.h

```diff
@@ -15,9 +15,11 @@ extern "C" {
 
 /** @brief Power domain selection */
 typedef enum __NRFS_PACKED {
-	GDPWR_POWER_DOMAIN_ACTIVE_FAST,
-	GDPWR_POWER_DOMAIN_ACTIVE_SLOW,
-	GDPWR_POWER_DOMAIN_MAIN_SLOW
+	GDPWR_GD_FAST_ACTIVE_0,
```

Comment:
These should have explicit numbers so it is easier to introduce changes and handle backwards compatibility